Description: Fix for bug"Float tags prevent opt() from finding later tags (KeyError instead of value of present tag)"
Origin: http://code.google.com/p/pysam/issues/detail?id=71

Index: samtools-0.1.18/bam.h
===================================================================
--- samtools-0.1.18.orig/bam.h	2012-12-21 12:00:47.293615029 -0500
+++ samtools-0.1.18/bam.h	2012-12-21 12:14:58.764541346 -0500
@@ -755,7 +755,7 @@
 {
 	if (x == 'C' || x == 'c' || x == 'A') return 1;
 	else if (x == 'S' || x == 's') return 2;
-	else if (x == 'I' || x == 'i' || x == 'f') return 4;
+	else if (x == 'I' || x == 'i' || x == 'f' || x == 'F') return 4;
 	else return 0;
 }
 
